doc/examples/contextual_enhancement.py

```diff
@@ -266,10 +267,12 @@
                         and Applications in Neuroimaging. Fifth International
                         Conference on Scale Space and Variational Methods in
                         Computer Vision
-.. [DuitsAndFranken_JMIV] R. Duits and E. Franken (2011) Morphological and
-                          Linear Scale Spaces for Fiber Enhancement in DWI-MRI.
-                          J Math Imaging Vis, 46(3):326-368.
-
+.. [DuitsAndFranken_JMIV] R. Duits and E. Franken (2011) Left-invariant diffusions 
```

Comment:
Different reference for the same statement?

You might want to change "DuitsAndFranken_JMIV" to reflect this change.